test/components/views/settings/tabs/room/AdvancedRoomSettingsTab-test.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+import { fireEvent, render, RenderResult } from "@testing-library/react";
+import { MatrixClient } from "matrix-js-sdk/src/client";
+import { Room } from "matrix-js-sdk/src/models/room";
+import { mocked } from "jest-mock";
+import { EventType, MatrixEvent } from "matrix-js-sdk/src/matrix";
+
+import AdvancedRoomSettingsTab from "../../../../../../src/components/views/settings/tabs/room/AdvancedRoomSettingsTab";
+import { mkEvent, mkStubRoom, stubClient } from "../../../../../test-utils";
+import dis from "../../../../../../src/dispatcher/dispatcher";
+import { Action } from "../../../../../../src/dispatcher/actions";
+import { MatrixClientPeg } from "../../../../../../src/MatrixClientPeg";
+
+jest.mock("../../../../../../src/dispatcher/dispatcher");
+
+describe("AdvancedRoomSettingsTab", () => {
+    const roomId = "!room:example.com";
+    let cli: MatrixClient;
+    let room: Room;
+
+    const renderTab = (): RenderResult => {
+        return render(<AdvancedRoomSettingsTab roomId={roomId} closeSettingsFn={jest.fn()} />);
+    };
+
+    beforeEach(() => {
+        stubClient();
+        cli = MatrixClientPeg.get();
+        room = mkStubRoom(roomId, "test room", cli);
+        mocked(cli.getRoom).mockReturnValue(room);
+    });
+
+    it("should render as expected", () => {
+        const tab = renderTab();
+        expect(tab.asFragment()).toMatchSnapshot();
+    });
+
+    it("should display room ID", () => {
+        const tab = renderTab();
+        tab.getByText(roomId);
```

Comment:
An `expect` is missing here ?

Reply:
The test fails if `getByText` fails, so I think this is OK? Is there something I'm missing?